app/web/typo3conf/ext/theme/Resources/Private/Partials/Page/Development.html

```diff
@@ -0,0 +1,7 @@
+{namespace s7=Sup7even\Theme\ViewHelpers}
+
+<f:if condition="{s7:context()} != 'Production'">
```

Comment:
that is just an example. it is on your own, how you want to apply that small div layer in your template. the main logic happens in the scss/css file with after/before things.